Mathlib.lean

```diff
@@ -3907,6 +3907,7 @@ import Mathlib.GroupTheory.Coxeter.Length
 import Mathlib.GroupTheory.Coxeter.Matrix
 import Mathlib.GroupTheory.Divisible
 import Mathlib.GroupTheory.DivisibleHull
+import Mathlib.GroupTheory.DivisibleHull2
```

Comment:
Why are there two of these files?! Why can't they be merged, or moved elsewhere?